api/orchestrator/orchestrator.proto

```diff
@@ -567,6 +575,34 @@ message ListCloudServicesResponse {
   string next_page_token = 2;
 }
 
+message CloudServiceStatisticsRequest {
+  optional string cloud_service_id = 1 [ (validate.rules).string.uuid = true ];
+}
+
+message CloudServiceStatisticsResponse {
+  // number of discovered resources per cloud services 
+  optional int64 number_of_discovered_resources = 1;
+  
+  // number of assessment results per cloud services 
+  optional int64 number_of_assessment_results = 2;
+  
+  // number of evidences per cloud services 
+  optional int64 number_of_evidences = 3;
+
+  // number of selected catalogs per cloud services 
+  optional int64 number_of_selected_catalogs = 4;
+
+  // number of available cloud services 
+  optional int64 number_of_cloud_services = 5;
```

Comment:
I think the number of metrics, catalogs and cloud service is not necessarily needed as a statistic because we are fetching them in the global layout of the page anyway because we need them in so many places.

If we want them, we might want to move them to a separate "global" statistics message?

Also I think we can get rid of the optionals. In the worst case, the number is just 0.

Reply:
I thought we can use the statistics endpoint for global and cloud service specific statistics. If a cloud service ID is specified, the global and specific statistics are returned, otherwise only the global statistics.

Reply:
> I thought we can use the statistics endpoint for global and cloud service specific statistics. If a cloud service ID is specified, the global and specific statistics are returned, otherwise only the global statistics.

Seems error-prone :) I would prefer two different end-points